pkg/html/releasehtml/failing_tests.go

```diff
@@ -11,6 +11,10 @@ import (
 )
 
 func summaryTopFailingTestsWithBug(topFailingTestsWithBug, allTests []sippyprocessingv1.FailingTestResult, numDays int, release string) string {
+	if len(topFailingTestsWithBug) == 0 {
+		return ""
```

Comment:
hm.  i think i'd prefer some sort of "this page(table) intentionally left blank" message over elliding it entirely.

Reply:
> hm. i think i'd prefer some sort of "this page(table) intentionally left blank" message over elliding it entirely.

It reads a little odd when wiring against dashboards that don't have bugzilla or issues wired.

Reply:
hm, fair enough.  but i reserve the right to revisit this if i start getting a lot of questions from build-watchers about blank sections :)

Reply:
> hm, fair enough. but i reserve the right to revisit this if i start getting a lot of questions from build-watchers about blank sections :)

Accepted, thanks.